doc/getting_started/getting_started_cartridge.rst

```diff
@@ -17,7 +17,7 @@ Next, create an application named ``myapp``. Say:
 
 This will create a Tarantool Cartridge application in the ``./myapp`` directory,
 with a handful of
-:ref:`template files and directories <creating-an-application-from-template>`
+:ref:`template files and directories <cartridge_cli_creating_an_application_from_template>`
```

Comment:
Sometimes it's `cartridge_cli_`, other times it's `cartridge-cli-`. What's the logic behind the label naming?

Reply:
Actually, more common usage is cartridge-cli-.... so I will just change this tag. 
We have some tag naming rules but here with cli I don't know how to follow them because it contains a dash in the name